content/introduction/_index.md

```diff
@@ -49,57 +53,60 @@ As mentioned, SODA Architecture is getting refined and optimized for different a
+  It is a docking station for heterogeneous storage backends! This is where all the different storage vendors’ drivers for various backends get attached.
+  More details [here](https://sodafoundation.io/projects/soda-dock/)
 
 - **SODA Delfin**
-It is storage infrastructure management solution to provide unified, intelligent and scalable resource management as well as alerting and performance monitoring of the underlying storage infrastructure. 
-More details [here](https://sodafoundation.io/projects/delfin-soda-infrastructure-manager/)
+  It is storage infrastructure management solution to provide unified, intelligent and scalable resource management as well as alerting and performance monitoring of the underlying storage infrastructure.
+  More details [here](https://sodafoundation.io/projects/delfin-soda-infrastructure-manager/)
 
 - **SODA Plugin**
-SODA North-Bound Plugin Project focuses to extend all the industry platforms and application solutions to interface with SODA API or be compliant with it.
-More details [here](https://sodafoundation.io/projects/soda-plugins/)
+  SODA North-Bound Plugin Project focuses to extend all the industry platforms and application solutions to interface with SODA API or be compliant with it.
+  More details [here](https://sodafoundation.io/projects/soda-plugins/)
 
 - **SODA Multicloud**
-SODA Multicloud project provides a cloud vendor agnostic data management for hybrid cloud, crosscloud or incloud. It can be hosted on prem or cloud native.
-More details [here](https://sodafoundation.io/projects/soda-multicloud-dds/)
+  SODA Multicloud project provides a cloud vendor agnostic data management for hybrid cloud, crosscloud or incloud. It can be hosted on prem or cloud native.
+  More details [here](https://sodafoundation.io/projects/soda-multicloud-dds/)
 
 - **SODA Orchestration**
-The Orchestration framework provides flexibility to use existing workflows or define customized workflows to get the simplified execution of tasks.
-More details [here](https://sodafoundation.io/projects/soda-experiments/)
-
+  The Orchestration framework provides flexibility to use existing workflows or define customized workflows to get the simplified execution of tasks.
+  More details [here](https://sodafoundation.io/projects/soda-experiments/)
 
 ## Official Charter
+
 The official charter for SODA Foundation under Linux Foundation can be found [here](https://sodafoundation.io/the-foundation/charter/)
 
 **Want to be part of this initiative?**
 Please join us to collaborate, research, develop and lead for unified data framework and standards.
 Contact us through any of the following channels:
 
- - [SODA Website](https://sodafoundation.io/the-foundation/join/)  
- - [SODA Slack](https://sodafoundation.io/slack)
- - [SODA Twitter](https://twitter.com/sodafoundation)  
- - [SODA Mailing List](https://lists.sodafoundation.io)
- - [SODA LinkedIn](https://www.linkedin.com/company/sodafoundation/)
+<ul style="list-style: none; ">
```

Comment:
Avoid using inline styles.
You can use classes and  add custom styles to this file `documentation/static/css/theme-opensds.css`
This is the custom theme file that is being used for the website.

Reply:
Thanks for the feedback! I'll make the changes you suggested to the theme-opensds.css file .